js/modules/k6/experimental/tracing/options.go

```diff
@@ -0,0 +1,40 @@
+package tracing
+
+import (
+	"errors"
+	"fmt"
+)
+
+// options are the options that can be passed to the
+// tracing.instrumentHTTP() method.
+type options struct {
+	// Propagation is the propagation format to use for the tracer.
+	Propagator string `js:"propagator"`
+
+	// Sampling is the sampling rate to use for the tracer.
+	Sampling *float64 `js:"sampling"`
+
+	// Baggage is a map of baggage items to add to the tracer.
+	Baggage map[string]string `js:"baggage"`
+}
+
+func (i *options) validate() error {
+	var (
+		isW3C    = i.Propagator == W3CPropagatorName
+		isB3     = i.Propagator == B3PropagatorName
+		isJaeger = i.Propagator == JaegerPropagatorName
+	)
+	if !isW3C && !isB3 && !isJaeger {
+		return fmt.Errorf("unknown propagator: %s", i.Propagator)
+	}
+
+	if i.Sampling != nil {
+		return errors.New("sampling is not yet supported")
```

Comment:
s/not yet supported/not supported/

I don't know the plan, but it sounds like a promise. Are we going to add it soon?

Reply:
Yes, I have a branch ready, and will open a PR as soon as the first chain is merged 👍🏻

Reply:
Can you add a to do for it, please?